client/landing/jetpack-cloud/components/log-item/docs/example.tsx

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
I'm assuming we're planning to integrate Jetpack Cloud components with devdocs and use that example there? Seems like a useful thing to work on in another PR.

Reply:
Yes, there's a ticket for this already: 1156570014567299-as-1159444362918521